Traceback (most recent call last):
  File "/home/runner/micromamba/envs/docs/lib/python3.12/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/runner/micromamba/envs/docs/lib/python3.12/site-packages/nbclient/client.py", line 1319, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/micromamba/envs/docs/lib/python3.12/site-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/micromamba/envs/docs/lib/python3.12/asyncio/base_events.py", line 691, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/home/runner/micromamba/envs/docs/lib/python3.12/site-packages/nbclient/client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "/home/runner/micromamba/envs/docs/lib/python3.12/site-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/runner/micromamba/envs/docs/lib/python3.12/site-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
barstar_F_over_E = msm.structure.align(barstar_F, selection='atom_name=="CA"',
                                       reference_molecular_system=barstar_E, reference_selection='atom_name=="CA"')
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mAttributeError[0m                            Traceback (most recent call last)
Cell [0;32mIn[12], line 1[0m
[0;32m----> 1[0m barstar_F_over_E [38;5;241m=[39m [43mmsm[49m[38;5;241;43m.[39;49m[43mstructure[49m[38;5;241;43m.[39;49m[43malign[49m(barstar_F, selection[38;5;241m=[39m[38;5;124m'[39m[38;5;124matom_name==[39m[38;5;124m"[39m[38;5;124mCA[39m[38;5;124m"[39m[38;5;124m'[39m,
[1;32m      2[0m                                        reference_molecular_system[38;5;241m=[39mbarstar_E, reference_selection[38;5;241m=[39m[38;5;124m'[39m[38;5;124matom_name==[39m[38;5;124m"[39m[38;5;124mCA[39m[38;5;124m"[39m[38;5;124m'[39m)

[0;31mAttributeError[0m: module 'molsysmt.structure' has no attribute 'align'

